smithy-cli/src/main/java/software/amazon/smithy/cli/commands/Upgrade1to2Command.java

```diff
@@ -321,6 +324,16 @@ public Void stringShape(StringShape shape) {
             return null;
         }
 
+        @Override
+        public Void listShape(ListShape shape) {
+            getDefault(shape);
+            if (shape instanceof SetShape) {
+                writer.updateLine(shape.getSourceLocation().getLine(),
+                    line -> line.replace("set", "@uniqueItems\nlist"));
```

Comment:
We need a test where there is "irregular" text / indentation before and after `set` on the same line, and where there are other occurrences of the letters `set` since [Java `String.replace()`](https://www.baeldung.com/string/replace) replaces all occurrences in a line, e.g.

Before:

```smithy
// "set" in "ResetResource"
resource ResetResource {
    properties: { member: String }
}

// 2 spaces, "set" in "@setter"
  @setter set for ResetResource {
    $member
}
```

After:

```smithy
// "set" in "ResetResource"
resource ResetResource {
    properties: { member: String }
}

// 2 spaces, "set" in "@setter"
  @setter @uniqueItems
list for ResetResource {
    $member
}
```

Maybe we need to add helper methods to `ModelWriter` for adding traits and renaming shapes based on source location?